definitions/ext-rf_scanner/definition.yml

```diff
@@ -21,20 +21,23 @@ synthesis:
   conditions:
     # The attribute’s value must match the provided value
     - attribute: eventType
+
       prefix: "THD_RF_TRANSACTION"
 
+
   # Telemetry attributes that should be extracted into entity tags.
   tags:
     ipAddr:
     osDetails:
+    macAddress:
+    serialNumber:
```

Comment:
Should these be multiValue: false? or we are expecting them to contain multiple values?
If so, how many are we thinking this will get?

Reply:
these do not need to be "multiValue: false" since they implicitly need to be unique per entity.